### PR TITLE
[chore] Mark attributes and metrics as experimental explicitly

### PR DESCRIPTION
**Pull request by lmolkova**
## Changes

In preparation for build-tools update (https://github.com/open-telemetry/build-tools/pull/272) we want to set stability explicitly on all implicit experimental attributes and metrics.

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] Change log entry added, according to the guidelines in [When to add a changelog entry](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md#when-to-add-a-changelog-entry).
  * If your PR does not need a change log, start the PR title with `[chore]`
* ~~[schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.~~

### CODE REVIEW

File: model/metrics/dotnet/dotnet-aspnetcore.yaml [hunk 1]
```diff
@@ -53,6 +58,7 @@ groups:
   - id: metric.aspnetcore.routing.match_attempts
     type: metric
     metric_name: aspnetcore.routing.match_attempts
+    stability: experimental
```
Comment (joaopgrassi) [inline model/metrics/dotnet/dotnet-aspnetcore.yaml:61]: Are these still experimental? Aren't they part of .NET 8 now? 🤔

File: model/metrics/dotnet/dotnet-kestrel.yaml [hunk 2]
```diff
@@ -14,6 +14,7 @@ groups:
   - id: metric.kestrel.active_connections
     type: metric
     metric_name: kestrel.active_connections
+    stability: experimental
```
Comment (joaopgrassi) [inline model/metrics/dotnet/dotnet-kestrel.yaml:17]: The document for these is marked as stable https://github.com/open-telemetry/semantic-conventions/blob/main/docs/dotnet/dotnet-kestrel-metrics.md

File: model/metrics/dotnet/dotnet-http.yaml [hunk 3]
```diff
@@ -2,6 +2,7 @@ groups:
   - id: metric.dotnet.dns.lookup.duration
     type: metric
     metric_name: dns.lookup.duration
+    stability: experimental
```
Comment (joaopgrassi) [inline model/metrics/dotnet/dotnet-http.yaml:5]: Same here, the markdown doc for these is marked as stable

File: model/metrics/dotnet/dotnet-signalr.yaml [hunk 4]
```diff
@@ -16,6 +16,7 @@ groups:
             - id: app_shutdown
               value: 'app_shutdown'
               brief: "The connection was closed because the app is shutting down."
+        stability: experimental
```
Comment (joaopgrassi) [inline model/metrics/dotnet/dotnet-signalr.yaml:19]: Same here, the markdown doc for these is marked as stable

File: model/scope/exporter/exporter.yaml [hunk 5]
```diff
@@ -6,10 +6,12 @@ groups:
     attributes:
       - id: name
         type: string
+        stability: experimental
```
Comment (joaopgrassi) [inline model/scope/exporter/exporter.yaml:9]: I think these should be stable, no? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/mapping-to-non-otlp.md

Comment (lmolkova) [reply model/scope/exporter/exporter.yaml:9]: I'm glad we're doing this explicitly :) 

Let me send separate PRs to add stability there

**Comment by lmolkova** [comment]
Thanks @joaopgrassi for the thorough review!

I took away .NET semconv from this PR - they are marked as stable, but refer to experimental attributes. I also got into some weird issues with current version of build-tools. I'll follow up on .NET separately.

So this PR should not have any controversial changes now and is ready for review again.

File: model/metrics/dotnet/dotnet-aspnetcore.yaml (hunk 1 above)
Comment (lmolkova) [reply model/metrics/dotnet/dotnet-aspnetcore.yaml]: see my comment below - will do .NET separately

**Review by trask — APPROVED** [review]

**Review by pyohannes — APPROVED** [review]

**Review by ChrsMark — APPROVED** [review]

**Review by arminru — APPROVED** [review]

**Comment by ChrsMark** [comment]
Some conflicts need to be resolved. @lmolkova please take care of those before merging.

**Comment by trisch-me** [comment]
I'm not opposed to making it explicit, but why not set the default stability level as `experimental`?

**Pull request merged**